spec/request/appeals_request_spec.rb

```diff
@@ -2,13 +2,75 @@
 
 require 'rails_helper'
 
-RSpec.describe 'Appeals Status', type: :request do
+RSpec.describe 'Appeals', type: :request do
   include SchemaMatchers
 
   before { sign_in_as(user) }
 
+  uuid = '1234567890'
+  ssn = '111223333'
+
+  %w[loa1 loa3].each do |user|
+    define_method("with_#{user}_user") do |options*|
+      using_ssn = options[:without_ssn] ? nil : ssn
+      let(:user) { FactoryBot.create(:user, user.to_sym, ssn: using_ssn) }
+    end
+  end
```

Comment:
This is hard to understand. It seems you could just define two `shared_contexts` which would read easier.

```
RSpec.shared_context "with_loa1_user" do
  let(:user) { ... }
end
```

Also, what does the `|options*|` splat usage do? I've never seen it used like so and can't find docs. Doesn't seem like you're passing it anywhere

Reply:
I converted this over to use `shared_context`, appreciate the suggestion. The splat wasn't necessary, it has been removed.